css/header.css

```diff
@@ -0,0 +1,269 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+body {
+	font-family: var(--main-font);
+}
+
+/* img {
+	display: block;
+	max-width: 100%;
+} */
+
+.img-container {
+	max-width: 390px;
+	/* for dev purpose only */
+}
+
+/* .block {
+	display: block;
+} */
+
+
+
+/* top bar section */
+.top-bar {
+	background-color: var(--color-onyx);
+	color: var(--color-white);
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	padding: 20px;
+}
+.dp-wrapper {
+width: var(--mobile-wrapper-width);
+display: flex;
+justify-content: space-between;
+}
+
+
+.logo-container {
+	display: flex;
+	margin-right: 65px;
+}
+
+/* desktop menu */
+.desktop-menu {
+	display: none;
+	flex-direction: row;
+	align-items: center;
+}
+
+.desktop-nav {
+	list-style-type: none;
+	text-decoration: none;
+	display: flex;
+	justify-content: space-between;
+	padding: 0;
+}
+
+.desktop-nav li {
+	padding: 10px 20px;
+}
+
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+a:hover,
+a:focus {
+	text-decoration: underline;
+	cursor: pointer;
+}
+
+a:visited {
+	color: inherit;
+}
+
+.button2 {
+	margin-left: 65px;
+	display: none;
+}
+
+/* mobile menu button style */
+#mobile-menu {
```

Comment:
The menu is still overlapping the button in some cases. The screenshot below is from a screen width of 890px.

Since `#nav-links-container` has  `z-index: 10;`, you will need give the button a higher z-index to `#mobile-menu` to put it on a higher layer: `z-index: 11;`

<img width="594" alt="Screen Shot 2024-01-22 at 12 30 28 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/5dc2310f-10c7-4be0-b7bc-2ee76283d52e">